css/hole.css

```diff
@@ -175,6 +175,7 @@ main > header nav {
 }
 
 #colors table, #morse table, #states table { break-inside: avoid-column }
+thead { user-select: none }
```

Comment:
What's this for? Should we do this for all table headers not just on the hole pages?

Reply:
On CSS colors, if I copy the big table into a text file, it looks like this:

```
...
Violet
#ee82ee
Orchid
#da70d6
Fuchsia
#ff00ff
Keyword	Hex         <--- this is annoying
Magenta
#ff00ff
MediumOrchid
#ba55d3
MediumPurple
#9370db
...
```